docs/content/direct/core-chart.md

```diff
@@ -181,6 +200,32 @@ After the initial installation is completed, there are two main ways to install
       --set-json='WDSes=[{name":"wds2"}]'
     ```
 
+The core chart also supports the use of external clusters as ITS Control Planes. A specific application is to connect to existing OCM clusters. As an example, create a first local kind cluster with OCM installed in it:
+
+```shell
+kind create cluster --name ext1
+
+clusteradm init
+```
+
+Then, create a second kind cluster suitable for KubeStellar installation and create a bootstrap secret in the new cluster with the kubeconfig information of the `ext1` cluster:
+
+```shell
+bash <(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/v$KUBESTELLAR_VERSION/scripts/create-kind-cluster-with-SSL-passthrough.sh) --name kubeflex --port 9443
+
+bash <(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/v$KUBESTELLAR_VERSION/scripts/create-external-bootstrap-secret.sh) --controlplane its1 --context kind-ext1 --address https://ext1-control-plane:6443 --verbose
+```
+
+Note the above command creates a secret named `its1-bootstrap` in the `default` namespace of the `kind-kubeflex` cluster. Finally, install the core chart using the `ext1` cluster as ITS:

```

Comment:
There are two commands, so I would say something like "Note that the last command above creates ...".